run.py

```diff
@@ -47,29 +69,68 @@ def process(message,firstname,username,chatid):
 		except Exception as e:
 			print("Error : 50 : "+str(e))
+
 # /help
 	elif "/help" in message[0]:
 		try:
-			sendMsg("@" + str(username) + " welcome. I'm the the Peak Shift @ Work Bot\n\nHere's how it works.\n\nYou can use @dogeshift_bot by messaging it directly or in a group that it is a part of.\n\nAvailable Commands\n\n/register - Registers your username with the bot\n/tip @username 10 doge - use this to tip some doge from your balance to another user\n/address - Get your deposit address\n/withdraw 100 <address> - to withdraw your balance",chatid)
+			sendMsg("Hey, so here's how this bot works.\n\nYou can use @dogeshift_bot by messaging it directly or in a group that it is a part of.\n\nAvailable Commands\n\n/register - Register Your Self In the bot\n/tip @username 10 doge - use this To Tip some doge from your balance to another user\n/address - Get Your Deposit Address\n/withdraw 100 <address> - to withdraw Your balance",chatid)
 		except Exception as e:
 			print("Error : 55 : "+str(e))
+
 # /register
 	elif "/register" in message[0]:
 		try:
-			block_io.get_new_address(label=username)
-			sendMsg("@"+username+" you are now registered.",chatid)
+			a = block_io.get_new_address(label=username+"00")
+			sendMsg("@"+username+" you are now registered.\nYour Address : <code>"+str(a['data']['address'])+"</code>",chatid)
 		except:
 			sendMsg("@"+username+" you are already registered.",chatid)
+
+# /work
+	elif "/work" in message[0]:
+		repos = ['peakshift/telegram-dogecoin']
+		g = Github(access_token)
+		msg1 = ""
+		for rep in repos:
+			repo = g.get_repo(rep)
+			open_issues = repo.get_issues()
+			a = 0
+			msg1 += "\n<b>"+str(repo.full_name)+"</b>\n\n"
+			for issue in open_issues:
+				a += 1
+				for x in issue.labels:
+					if x.name == "reward":
+						open_issues = repo.get_issue(number=int(issue.number))
+						one = md.parse(open_issues.body)
+						ym = yaml.safe_load(one[0].content)
+						ymm = json.loads(json.dumps(ym, default = myconverter))
+						msg1 += "<code>#"+str(issue.number)+" > "+str(issue.title)+"</code> -- "+str(ym['Reward'])+" 💰\n"
+		msg1 += "\n\n-> Use /body_[issue_number] , Example : <code>/body_13</code>"
+		sendMsg(msg1,chatid)
+
+# /body
+	elif "/body" in message[0]:
+		spl = str(message[0]).split("_")
+		if spl[1] != None:
+			#sendMsg(spl[1],chatid)
+			g = Github(access_token)
+			repo = g.get_repo('peakshift/telegram-dogecoin')
+			open_issues = repo.get_issue(number=int(spl[1]))
+			#one = md.parse(open_issues.body)
+			sendMsg(open_issues.body,chatid,"markdown")
+		else:
+			sendMsg("null",chatid)
+
 # /balance
 	elif "/balance" in message[0]:
 		try:
 			(balance, pending_balance) = returnBal(username)
-			sendMsg("@"+username+" Balance : "+balance+ " Doge ("+pending_balance+" Doge)",chatid)
+			sendMsg("@"+username+" Balance : "+balance+ "Doge ("+pending_balance+" Doge)",chatid)
```

Comment:
```suggestion
			sendMsg("@"+username+" Balance : "+balance+ " Doge ("+pending_balance+" Doge)",chatid)
```
There are a bunch of changes that are overwritten in this PR. This spacing was fixed in a previous commit for example. Please compare the code in the `master` branch.